lua/neogit/status.lua

```diff
@@ -611,6 +611,8 @@ local function get_selected_commits()
   return items
 end
 
+M.get_selectet_commits = get_selected_commits
```

Comment:
Can we just expose this at the function definition instead of here? It makes the goto definition work better that way